aio_cooker.simba

```diff
@@ -196,14 +207,16 @@ function TCooker.HandleMake: Boolean;
 begin
   if not Make.IsOpen then
     Exit;
-
+  if 'karambwan' in Food then
+  Cooking := Result := Make.Select(1, MAKE_QUANTITY_ALL, SRL.Dice(98.5))
+    else
   Cooking := Result := Make.Select(0, MAKE_QUANTITY_ALL, SRL.Dice(98.5));
 end;
 
 function TCooker.WaitCooking: Boolean;
 begin
   if Cooking then
-    Result := Cooking := Self.HasIngredient and XPDrop.WaitDrop(2500)
+    Result := Cooking := Self.HasIngredient and XPDrop.WaitDrop(15500)
```

Comment:
Why 15500 seconds? it should be the time it takes to cook per item.
Otherwise if something goes wrong and you stop cooking you will be stopped 15 seconds without doing anything.

Reply:
XP bar doesn't show when you burn food, I was burning a lot in the beginning. At 2500 it starts cooking again every time you burn a food item